services/121-service/examples/fsp-anonymous2.json

```diff
@@ -0,0 +1,13 @@
+{
+  "fsp": "Mobile Money Provider B",
+  "attributes": [
+    {
+      "name": "phoneNumber",
+      "label": {
+        "en": "What is your phonenumber, so we can send you mobile money"
+      },
+      "answerType": "text",
```

Comment:
We could use `tel`;
So that we can add some additional features to the input-field (like syntax-checking etc)

Reply:
Cool i'll change that